registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/RegistrationCheckerCompilerPlugin.scala

```diff
@@ -20,6 +20,9 @@ class RegistrationCheckerCompilerPlugin(override val global: Global) extends Plu
   override val components: List[PluginComponent] = List(classSweep, serializerCheck)
 
   override def init(options: List[String], error: String => Unit): Boolean = {
+    if (options.contains("--disable"))
```

Comment:
Actually... the option is only applicable to this compiler plugin but not to to class sweeper? 🤔

Reply:
This option disables all of the compiler plugin, which means `PluginComponents` are never consumed by the compiler and new phases are never created.

Reply:
From `def init` documentation:

> The plugin can opt out of further processing by returning false.  For example, if the plugin has an "enable" flag, now would be a good time to sit on the bench.

Reply:
Aww okay I forgot there is single compiler plugin with two components (rather than two compiler plugins) 🤦🏻 nvm